src/multiprocess_wifi_listener.py

```diff
@@ -15,8 +15,10 @@ def wifi_listener(wifi_card, queue):
     :param queue:
     :return:
     """
-    for frame in pyshark.LiveCapture(interface=wifi_card, monitor_mode=True):
-        queue.put(WifiFrame.from_frame(frame))
+    print("Starting listener on {}".format(wifi_card_name))
```

Comment:
Går ud fra at dette er et produktions relateret print

Reply:
Synes det er en fin info, dont you?